detect_secrets/core/secrets_collection.py

```diff
@@ -200,7 +201,8 @@ def scan_file(self, filename, filename_key=None):
 
         if os.path.islink(filename):
             return False
-
+        if os.path.splitext(filename)[1] in IGNORED_FILE_EXTENSIONS:
```

Comment:
This turned out to be a sneaky bug I found today, `splitext` will always have `.extension` in it, and so we were checking if e.g. `.svg` == `svg`. I'll push a fix 👍

Reply:
Hmm, we did the same thing [in -server](https://github.com/Yelp/detect-secrets-server/blob/618b251919f6834deab9c78ffeb4a506516e79cc/detect_secrets_server/storage/core/git.py#L156) too